src/components/UpdatesFeed/UpdateCard.test.js

```diff
@@ -0,0 +1,104 @@
+import { render, screen } from "@testing-library/react";
+
+import UpdateCard from "./UpdateCard";
+
+describe("UpdateCard tests", () => {
+	let mockProject, viewDocumentButtonName, viewPcpButtonName, viewProjectButtonName;
+
+	beforeEach(() => {
+		mockProject = {
+			_id: "1",
+			location: "Location",
+			name: "Project 1",
+			region: "Region",
+			type: "type",
+		};
+		viewDocumentButtonName = "View Document(s)";
+		viewPcpButtonName = "View Engagement";
+		viewProjectButtonName = "View Project";
+	});
+
+	describe("when the update has no documents or PCPs", () => {
+		test("should render the UpdateCard correctly", () => {
+			const mockUpdate = {
+				documentUrl: "",
+				project: mockProject,
+				updateContent: "Update content.",
+				updateDate: "2022-09-13T23:03:02.878Z",
+				updateName: "Update Name",
+			};
+
+			render(<UpdateCard {...mockUpdate} />);
+
+			const documentButton = screen.queryByRole("button", { name: viewDocumentButtonName });
+			const pcpButton = screen.queryByRole("button", { name: viewPcpButtonName });
+			const projectLink = screen.getByText(viewProjectButtonName).closest("a");
```

Comment:
Can't you use `screen.getByRole("link", { name: viewProjectButtonName })` here? That's preferred.

Also, please group your `const` variables with their assertions. Eg.

```
const a = screen.getByRole(...);
expect(a).toBeInTheDocument();
expect(a).toHaveAttribute("href", <some-path>);

const b = screen.getByRole(...);
...
```

Reply:
updated in e791791